conanfile.py

```diff
@@ -80,8 +80,7 @@ def package(self):
                 self.copy(pattern="*zlib.lib", dst="lib", src="_build", keep_path=False)
                 self.copy(pattern="*zlib.dll.a", dst="lib", src="_build", keep_path=False)
             else:
-                self.copy(pattern="*zlibstaticd.*", dst="lib", src="_build", keep_path=False)
-                self.copy(pattern="*zlibstatic.*", dst="lib", src="_build", keep_path=False)
+                self.copy(pattern="zlibstatic*.lib", dst="lib", src="_build/lib")
```

Comment:
Use only `_build` as `src`. No `_build/lib`.
```suggestion
                self.copy(pattern="zlibstatic*.lib", dst="lib", src="_build")
```